controllers/utils/env.go

```diff
@@ -0,0 +1,28 @@
+package utils
+
+import (
+	"os"
+
+	"github.com/joho/godotenv"
+)
+
+func GetEnv(key string, filenames ...string) string {
+	if value, ok := os.LookupEnv(key); ok {
+		return value
+	}
+	envMap, err := godotenv.Read(filenames...)
```

Comment:
Also to check if filenames actually exist otherwise return empty after checking globally

Reply:
Good catch. Will fix it.